systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1099,11 +1100,7 @@ void testTopicWithoutLabels() {
         KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3).done();
 
         // Creating topic without any label
-        KafkaTopicResource.topic(CLUSTER_NAME, "topic-without-labels")
-            .editMetadata()
-                .withLabels(null)
-            .endMetadata()
-            .done();
+        KafkaTopicResource.topicWithoutWait(CLUSTER_NAME, "topic-without-labels", null);
```

Comment:
Why did we needed a new method for this when it seems to be used only here? Why not just do here:

```java
KafkaTopicResource .topicWithoutWait(defaultTopic(clusterName, topicName, 1, 1)
    .editMetadata()
        .withLabels(labels)
    .endMetadata()
    .build());
```